ui/selectors/multichain.ts

```diff
@@ -671,3 +721,13 @@ export function getLastSelectedSolanaAccount(state: MultichainState) {
     );
   return sortedNonEvmAccounts.length > 0 ? sortedNonEvmAccounts[0] : undefined;
 }
+
+export function getLastSelectedTronAccount(state: MultichainState) {
+  const nonEvmAccounts = getInternalAccounts(state);
+  const sortedNonEvmAccounts = nonEvmAccounts
+    .filter((account) => isTronAccount(account))
+    .sort(
+      (a, b) => (b.metadata.lastSelected ?? 0) - (a.metadata.lastSelected ?? 0),
+    );
+  return sortedNonEvmAccounts.length > 0 ? sortedNonEvmAccounts[0] : undefined;
+}
```

Comment:
Note for other reviewers: This function is added since we have similar functions Bitcoin and Solana, but with the new account model (BIP-44) we should not rely on the `AccountsController` internals anymore. This kind of logic should use account grounds and not use "single accounts" anymore.

I'll make a note to remove/cleanup those functions later on, for now we can keep it (like the others)